csfieldguide/templates/interactives/short-term-memory-picture.html

```diff
@@ -0,0 +1,125 @@
+          </div>
+          <div id="item-duck" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-duck' %}
+          </div>
+          <div id="item-guitar" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-guitar' %}
+          </div>
+
+          <div id="item-bike" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-bicycle' %}
+          </div>
+          <div id="item-camera" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-camera' %}
+          </div>
+          <div id="item-soccerball" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-soccer-ball' %}
+          </div>
+
+          <div id="item-book" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-book' %}
+          </div>
+          <div id="item-toothbrush" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-electric-toothbrush' %}
+          </div>
+          <div id="item-teapot" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-teapot' %}
+          </div>
+          <div id="item-calculator" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-calculator' %}
+          </div>
+
+          <div id="item-forklift" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-fork-lift' %}
+          </div>
+          <div id="item-sock" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-socks' %}
+          </div>
+          <div id="item-cactus" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-cactus' %}
+          </div>
+
+          <div id="item-cup" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-cup' %}
+          </div>
+          <div id="item-envelope" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-envelope' %}
+          </div>
+          <div id="item-glasses" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-glasses' %}
+          </div>
+          <div id="item-backpack" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-school-backpack' %}
+          </div>
+
+          <div id="item-tshirt" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-t-shirt' %}
+          </div>
+          <div id="item-chair" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-chair' %}
+          </div>
+          <div id="item-sheep" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-sheep' %}
+          </div>
+        </div>
+      </div>
+
+      <div id="answer-input" class="col-12 text-center d-none">
+        <div class="row justify-content-center">
+          <p>{% trans "Enter the items you remember below! <span id='stm-instructions'>Please separate each item with a comma.</span>" %}</p>
```

Comment:
Would be great if items could also be separated by a newline - It's another very intuitive way of making a list